corehq/apps/domain/deletion.py

```diff
@@ -210,6 +210,10 @@ def _delete_custom_data_fields(domain_name):
     CustomDeletion('form_processor', _delete_all_cases),
     CustomDeletion('form_processor', _delete_all_forms),
     ModelDeletion('aggregate_ucrs', 'AggregateTableDefinition', 'domain'),
+    ModelDeletion('app_manager', 'AppReleaseByLocation', 'domain'),
```

Comment:
what I am understanding is this will delete the all records from `AppReleaseByLocation` model (which is part of `app_manager` app) where domain='domain_being_deleted'

Reply:
yep, exactly